pkg/awsutils/awsutils.go

```diff
@@ -1296,6 +1318,38 @@ func (cache *EC2InstanceMetadataCache) cleanUpLeakedENIs() {
 	}
 }
 
+func (cache *EC2InstanceMetadataCache) tagENIcreateTS(eniID string, maxBackoffDelay time.Duration) {
+	// Tag the ENI with "node.k8s.amazonaws.com/createdAt=currentTime"
+	tags := []*ec2.Tag{
+		{
+			Key:   aws.String(eniCreatedAtTagKey),
+			Value: aws.String(time.Now().Format(time.RFC3339)),
+		},
+	}
+
+	log.Debugf("Tag untagged ENI %s: key=%s, value=%s", eniID, aws.StringValue(tags[0].Key), aws.StringValue(tags[0].Value))
+
+	input := &ec2.CreateTagsInput{
+		Resources: []*string{
+			aws.String(eniID),
+		},
+		Tags: tags,
+	}
+
+	_ = retry.RetryNWithBackoff(retry.NewSimpleBackoff(500*time.Millisecond, maxBackoffDelay, 0.3, 2), 5, func() error {
+		start := time.Now()
+		_, err := cache.ec2SVC.CreateTagsWithContext(context.Background(), input, userAgent)
+		awsAPILatency.WithLabelValues("CreateTags", fmt.Sprint(err != nil)).Observe(msSince(start))
+		if err != nil {
+			awsAPIErrInc("CreateTags", err)
+			log.Warnf("Failed to tag the newly created ENI %s:", eniID)
```

Comment:
Nit: Log message is not right, and won't log the error. 

Maybe `log.Warnf("Failed to add tag to ENI %s: %v", eniID, err)`